demo/src/main/java/com/google/googlemediaframeworkdemo/demo/adplayer/ImaPlayer.java

```diff
@@ -55,6 +55,8 @@
  */
 public class ImaPlayer {
 
+  private static String PLAYER_TYPE = "GMF ImaPlayer";
```

Comment:
The example code uses "'google/gmf-player'" as the id. I would advocate for google/gmf-android and you should also set the version if possible.